lua/nvim-tree/git/utils.lua

```diff
@@ -41,7 +41,7 @@ function M.should_show_untracked(cwd)
     return untracked[cwd]
   end
 
-  local cmd = "git -C " .. cwd .. " config --type=bool status.showUntrackedFiles"
+  local cmd = "git -C " .. cwd .. " config status.showUntrackedFiles"
```

Comment:
Thanks for finding this. It looks like I only tested "no" and nil which are the only values that map to a bool.